### PR TITLE
feat(eam): add `EAM#UpdateDeployers`.

### PR DESCRIPTION
**Pull request by raulk**
- This method allows updating a deployer allowlist.
- It can only be called when the permissioning mode is Allowlist.
- It must be called by a currently authorized deployer.

This method is bound to an FRC42 method number. Its arguments are CBOR-encoded. It can be called from an f1/f3 account by sending a native transaction, or from an f410 account using this path: https://github.com/consensus-shipyard/ipc/pull/773.

### CODE REVIEW

File: fendermint/actors/eam/src/lib.rs [hunk 1]
```diff
@@ -59,6 +68,33 @@ impl IPCEamActor {
 
         Ok(())
     }
+
+    fn update_deployers(rt: &impl Runtime, deployers: Vec<Address>) -> Result<(), ActorError> {
```
Comment (cryptoAtwill) [inline fendermint/actors/eam/src/lib.rs:72]: do we do checks on the deployers? Like simple dedup, size limiting?

Comment (cryptoAtwill) [reply fendermint/actors/eam/src/lib.rs:72]: I think maybe we should restrict to size at least 1 so that we still have someone to update the deployers.

Comment (cryptoAtwill) [reply fendermint/actors/eam/src/lib.rs:72]: oh yeah, it's a map, so no need to do dedup.

Comment (raulk) [reply fendermint/actors/eam/src/lib.rs:72]: On 0 check: that would make it impossible to remove all deployment privileges, which is a legitimate use case.

**Review by fridrik01 — APPROVED** [review]
lgtm, but there are some lint errors

**Pull request merged**